protocol/app/app.go

```diff
@@ -529,6 +536,19 @@ func New(
 		scopedIBCKeeper,
 	)
 
+	// Create ICA Host Keeper
+	app.ICAHostKeeper = icahostkeeper.NewKeeper(
+		appCodec,
+		keys[icahosttypes.StoreKey], // key
+		app.getSubspace(icahosttypes.SubModuleName), // paramSpace
+		app.IBCKeeper.ChannelKeeper,                 // ics4Wrapper, may be replaced with middleware such as ics29 fee
+		app.IBCKeeper.ChannelKeeper,                 // channelKeeper
+		&app.IBCKeeper.PortKeeper,                   // portKeeper
```

Comment:
for my own understanding, why do we need to pass the pointer?

Reply:
This method in `ibc-go` portKeeper has a pointer receiver (not sure if intended or not), so a pointer instead of a copy needs to passed here